stripe/res/values/strings.xml

```diff
@@ -118,4 +118,5 @@
     <!--Text informing user they should add a debit or credit card. Shown when there are no existing payment methods in the application-->
     <string name="add_card">Add a new debit or credit card to make purchases in this app.</string>
     <string name="invalid_shipping_information">Invalid Shipping Address</string>
+    <string name="pay_with_google" translatable="false">Pay with Google</string>
```

Comment:
remove `translatable="false"`

Reply:
cool + translations (I hope i got the Japanese right. The Singaporean fellow we have here only speaks Mandarin, which helps with chinese...but his Japanese is not great, so couldn't confirm google translate)